### ISSUE TITLE
Resetting body causes infinite loop

### ISSUE THREAD

**Issue by emmamei** [emmamei/cdkey#719]
When the body is reset, an infinite loop is generated:
```
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carrierID [Avatar] = 00000000-0000-0000-0000-000000000000
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carrierName [Avatar] = 
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carryExpire [Avatar] = 0
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carryExpire [Main] = 0
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carrierID [Avatar] = 00000000-0000-0000-0000-000000000000
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carrierName [Avatar] = 
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carryExpire [Avatar] = 0
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carryExpire [Main] = 0
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carrierID [Avatar] = 00000000-0000-0000-0000-000000000000
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carrierName [Avatar] = 
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carryExpire [Avatar] = 0
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carryExpire [Main] = 0
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carrierID [Avatar] = 00000000-0000-0000-0000-000000000000
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carrierName [Avatar] = 
[11:11:15] [linked] Doll Key: DEBUG-LINKMONITOR(8):179: Link message #SEND_CONFIG cmd: carryExpire [Avatar] = 0
```
Probably one of those `SEND_CONFIG` entries is generating a loop. It appears as if Avatar.lsl and Main.lsl main be interacting.

**Comment by emmamei** [comment]
Retrying the Reset Body function failed differently this time in that it never found the `~normaloutfit` folder nor did it attach it and detach the rest...

**Comment by emmamei** [comment]
This would appear to have been fixed 11 days ago with the commit 1b8c4e2078c7a49c77e2ca9af7e49ba22dd49c32 - which made it so that Main.lsl only sends out `carryExpire` if the carrier is missing.

Perhaps there is a problem with that, but seems unlikely.

**Comment by emmamei** [comment]
The files do not account for the reporting above, and recent tests do not show these results. Closing without fix.
